XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -31,7 +31,6 @@
     <PackageProjectUrl>https://github.com/xamarin/XamarinCommunityToolkit</PackageProjectUrl>
     <DebugType>portable</DebugType>
     <Configurations>Debug;Release</Configurations>
-    <LangVersion>8.0</LangVersion>
```

Comment:
Why do we delete this line?)

Reply:
It's moved into `Directory.Build.props` it seems?